server/proxy/proxy.go

```diff
@@ -86,6 +88,45 @@ func (p *Proxy) Notify(cc *apps.Context, subj apps.Subject) error {
 	return nil
 }
 
+func (p *Proxy) NotifyRemoteWebhook(app *apps.App, data []byte, webhookPath string) error {
+	if !app.GrantedPermissions.Contains(apps.PermissionRemoteWebhooks) {
+		return utils.NewForbiddenError("%s does not have permission %s", app.AppID, apps.PermissionRemoteWebhooks)
+	}
+
+	up, err := p.upstreamForApp(app)
+	if err != nil {
+		return err
+	}
+
+	var datav interface{}
+	err = json.Unmarshal(data, &datav)
+	if err != nil {
+		// if the data can not be decoded as JSON, send it "as is", as a string.
+		datav = string(data)
+	}
```

Comment:
What value does that unmarshaling the JSON add? Does it make the developers life easier?